notebooks/book1/02/gamma_dist_plot.ipynb

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.            plt.plot(x, y, p, lw=2, label="a=%.1f,b=%.1f" % (a, b))
</pre><p>I think using <code>lw=2</code> making line too much thick in latexified plot, what do you think? We can use defaults may be</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Default line width looks neater . Have updated the code